### PR TITLE
@eessex: Artsy Primer article layout

### PR DESCRIPTION
**Pull request by craigspaeth**
This adds a layout for articles we're going to publish as a part of our "New Collector Project" which is an experiment to test converting ad spend to a new collector offering. These articles will play the part of acting like high-end landing pages with artwork recommendations that Sarah G. can customize to each collector who converts. As such, we want to make sure it has a stripped down layout so the user doesn't bounce to related articles or otherwise. I'll comment on refactors more inline.

image

### CODE REVIEW

File: apps/artsy_primer/routes.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Article from '../../models/article'
+import embed from 'particle'
+import { crop } from '../../components/resizer'
+
+export const article = async (req, res, next) => {
+  const article = new Article({id: req.params.id})
+  await article.fetchWithRelated({
```
Comment (craigspaeth) [inline apps/artsy_primer/routes.js]: Using ES6 here which gives us [async/await](https://www.twilio.com/blog/2015/10/asyncawait-the-hero-javascript-deserved.html) ❤️

File: assets/article.styl [hunk 2]
```diff
@@ -1,15 +1 @@
...
-@require '../components/merry_go_round'
-@require '../components/article_figure'
-@require '../components/article/stylesheets'
-@require '../components/grid'
-@require '../components/cta_bar'
-@require '../components/gradient_blurb'
-@require '../components/jump'
-@require '../components/email/stylesheets'
-@require '../components/articles_grid/stylesheets'
 @require '../apps/article/stylesheets'
```
Comment (craigspaeth) [inline assets/article.styl]: Once upon a time Stylus didn't have a `@require` feature that would automatically know how to de-duplicate `@include`s. Now that we're using `@require` we can better modularize our stylus files in the same way we expect `require`s to work with Browserify. Hence why I'm moving these requires out of the "entry point" of this asset file and into the components and apps themself.

File: apps/artsy_primer/templates/article.jade [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        h2.artsy-primer-h2 Portfolio
+        h3.artsy-primer-h3 Your personalized collection of available artwork.
+        .artsy-primer-user
+          include ../../../components/main_layout/header/templates/user
+
+  .artsy-primer-article
+    include ../../../components/article/templates/index
+
+  .responsive-layout-container: .mlf.mlf__fullwidth
+    include ../../../components/main_layout/footer/lower
```
Comment (craigspaeth) [inline apps/artsy_primer/templates/article.jade]: Below you can see that I've broken up the main layout templates so I can more easily re-use the individual pieces to construct a stripped down form of it in the screenshot embedded in the description.

File: assets/article.styl (hunk 2 above)
Comment (eessex) [inline assets/article.styl]: Good call!  Glad to know these differences with require and include

**Comment by eessex** [comment]
This all looks good to me, feel free to merge when 🍏

**Pull request merged**